src/main/java/reposense/wizard/prompt/ViewPrompt.java

```diff
@@ -0,0 +1,30 @@
+package reposense.wizard.prompt;
+
+import java.nio.file.Paths;
+
+import reposense.wizard.InputBuilder;
+
+/**
+ * Represents a Prompt to get the view flag.
+ */
+public class ViewPrompt extends Prompt {
+    public static final String DESCRIPTION = "Enter directory of report to display in server. "
+            + "If not specified, default directory will be used";
+
+    public ViewPrompt() {
+        super(DESCRIPTION);
+    }
+
+    @Override
+    public InputBuilder addToInput(InputBuilder inputBuilder) {
+        if (getResponse().isEmpty()) {
+            return inputBuilder.addView();
+        }
+        return inputBuilder.addView(Paths.get(getResponse()));
+    }
+
+    @Override
+    public Prompt[] run() {
+        return new Prompt[] {};
+    }
```

Comment:
I don't think this is correct. It should be view or no view without specification. The `-v` with argument is for when the user does not want to analyze anything and just wants to view a pre-analyzed report

Reply:
Got it, we will clean up the unused `InputBuilder.java` functions.

Reply:
The logic for this is implemented in `OptionalViewPrompt.java`, where the user is prompted with "Do you want to start a server to display the report?", allowing them to choose whether or not to use the `ViewPrompt`. `ViewPrompt` currently contains logic for configuration of the `--view` flag if it is present.

Reply:
I understand that. But the OptionalViewPrompt shouldn't spawn a ViewPrompt since they dont serve the same purpose. There is no need for the user to supply an argument for --view unless they simply want to view a report that has already been generated, so it doesn't really make sense at all in this wizard walkthrough

Reply:
Yes, for this I believe it will be better to drop the need for an argument as there is no way of changing the output directory immediately after generating it anyway.